src/app/pages/news/9.3/04-role-welcome.md

```diff
@@ -1,10 +1,14 @@
 ## Role based Welcome {#role93}
 
-- role based designer experience
+The Designer welcome page assists you in settings up the workspace for your needs.
+
+- **Accessible**: simple and uncluttered featureset to onboard new users quickly into the first Process drawing experience.
+- **Advanced actions**: Testers and Developers don't loose any features, but gain quick access to advanced features.
+- **Project filters**: contents within the Project tree will be filtered according to your role. Either to reduce complexity or to expose advanced configs. You can switch at any time to a different filterset.
 
 <div class="short-links">
-	<a href="/portal/9.2/doc/portal-developer-guide/introduction/index.html#new-noteworthy-9-2"
+	<a href="${docBaseUrl}/designer-guide/process-modeling/projects/ivyProjectView.html"
 		target="_blank" rel="noopener noreferrer">
-		<i class="si si-book"></i> Portal New & Noteworthy
+		<i class="si si-book"></i> Designer Guide
 	</a>
 </div>
```

Comment:
Top! but I would show the xNewWelcomePage.png as first "big" image

Reply:
Thanks @ivy-bb , I've adjust the the picture order as suggested